### DISCUSSION TITLE
Pending speech to text; high latencies

### DISCUSSION THREAD

**Discussion by itstheraj** [deepgram/community#710] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

As of the morning of Wednesday April 17th from 7 AM PST we are noticing 10x higher latency on the /listen endpoint. Deepgram web console is loading much slower and the logs are showing many POST requests stuck in pending which we have never seen. Is this a degraded performance issue?

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?smart_format=true&punctuate=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

bb986476-cef4-4e76-8937-d5e1f3afc855

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by db-aeon** [comment]
Same here -random timeouts or empty responses...

**Comment by jpvajda** [comment]
It's quite possible what you both experienced were delayed response from The Deepgram Services which can happen from time to time. You can track [the status here](https://status.deepgram.com/) in the future to get a real time sense of down times or delays in our APIs. 

Are you still seeing delay issues @db-aeon @itstheraj ?

**Reply by itstheraj** [reply]
Currently no, but the status page did not reflect any degradation in performance at all, and no incident report at the time of the increased latency.

I'm not sure what your observability stack looks like but you may want to consider alerting and declaring incident on anomalous outbound latency - given that you all provide a fairly compute heavy service it may help set expectations with service clients :)

Thank you for responding and following up!

**Comment by MikeZyatkov** [comment]
I've been experiencing the same issue for the last week. Around 9 pm GTM the latency from /listen API endpoint is increasing around x10 (up to 30 - 35 seconds)

**Comment by jacksonhmg** [comment]
Exact same experience. The latency has been massive recently, and the status page does not reflect it. Only have noticed this problem in the last week or so.

**Comment by jpvajda** [comment, marked as answer]
@jacksonhmg  @MikeZyatkov @itstheraj  Here's what I learned by discussing these concerns with our Engineering Teams

We've been experience higher than average loads on our` /listen` endpoint this past week an are currently working on scaling to support that additional traffic.

We apologize for any inconvenience this is causing and we are working to address these concerns.

If you want to provide specific request IDs we can looking into those requests for more details.

**Reply by jacksonhmg** [reply]
Thanks for the update. Please notify me once the scaling has been improved, I'm eager to get back to using Deepgram once it is
